x/ccv/provider/types/events.go

```diff
@@ -17,6 +21,13 @@ const (
 	AttributeConsumerConsensusPubKey   = "consumer_consensus_pub_key"
 	AttributeAddConsumerRewardDenom    = "add_consumer_reward_denom"
 	AttributeRemoveConsumerRewardDenom = "remove_consumer_reward_denom"
+	AttributeSubmitterAddress          = "submitter_address"
 	AttributeConsumerCommissionRate    = "consumer_commission_rate"
-	AttributeConsumerId                = "consumer_chain_id"
+	AttributeConsumerID                = "consumer_id"
```

Comment:
Can we use ... `ConsumerId` instead of `ConsumerID` and similarly for `ChainID`. I feel we are inconsistent in general in the naming. I personally prefer the `Id` instead of the `ID` notation. If this is a linter issue, could we fix it there (maybe in other PR)?